contracts/PodCastNFT.sol

```diff
@@ -224,4 +224,11 @@ contract PodCastNFT is ERC721URIStorage, Ownable {
         bool success = henkakuToken.transfer(fundAddress, _amount);
         require(success, "Transaction Unsuccessful");
     }
+
+    function burn(
+        uint256 _tokenId
+    ) public onlyOwner {
+        require(getRoles(ownerOf(_tokenId)).length == 0, "membership nft is still owned by member");
```

Comment:
[IMO] I write it in there just in case to prevent mishandling.

Reply:
as far as I understand when we burn nft. just call burn is enough.
if we need to delete the roles then we should write down the setEmptyRole method.

Reply:
I understand. I was thinking it might be unnecessary, so I'll remove it.

Reply:
fixed in https://github.com/henkaku-center/podcast-nft/pull/33/commits/93adf89a6766a8579746ff6c324c62357fd82e5d